perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -3689,6 +3689,7 @@ public boolean removeAttributeWithoutCheck(PerunSession sess, String key, Attrib
 			} catch (WrongAttributeValueException | WrongReferenceAttributeValueException ex) {
 				throw new InternalErrorException(ex);
 			}
+			log.info("{} removed attribute: {} by key = {}",sess.getPerunClient().getId(), attribute.getName(), key);
```

Comment:
``PerunClient`` is valid only for OAUTH2 clients. Otherwise its half empty object, especially client ID is not present. I would prefer logging whole ``PerunSession``, but I understand, that this message should be rather short. But neither ``PerunSession`` or ``PerunPrincipal`` within it have a short ``toString()``. Please add method in ``PerunPrincipal`` like: ``getLogId()`` which will return value of ``actor`` property, so we will see user identity. If it won't be sufficient in a future, we can safely change such method to return also extSourceName or other properties. Thank you.